### DISCUSSION TITLE
Accordion Item open by default?

### DISCUSSION THREAD

**Discussion by thedittmer** [huntabyte/shadcn-svelte#471] [General]
I'm trying to make an accordion item open by default. I've looked at the Bits UI documentation and I found that I can set the data-state attribute to open. However, I'm not able to make this work. Please let me know if you have an idea that might help me.
https://www.bits-ui.com/docs/components/accordion

**Comment by huntabyte** [comment]
Just set the value of the `Accordion.Root` to the same value of the item you wish to be open by default!

**Reply by thedittmer** [reply]
That worked!! Thank you!
